### PR TITLE
[5.2] Fix increment on non-alphanumeric string deprecation

### PR DESCRIPTION
**Pull request by Quy**
Pull Request for Issue #43896 .

### Summary of Changes
Fix increment on non-alphanumeric string deprecation on php 8.3.

### Testing Instructions
Enable Error Reporting to Maximum.
Create/save an article.
Create/save a banner.
Create/save a contact.
Create/save a newsfeed.
Create/save a tag.
See PHP error log.
Apply PR.
Under Publishing tab, check Revision value incrementing.

### Actual result BEFORE applying this Pull Request
PHP Deprecated: Increment on non-alphanumeric string is deprecated

### Expected result AFTER applying this Pull Request
No error messages.

### CODE REVIEW

File: administrator/components/com_content/src/Model/ArticleModel.php [hunk 1]
```diff
@@ -346,7 +346,11 @@ protected function prepareTable($table)
         }
 
         // Increment the content version number.
-        $table->version++;
+        if (empty($table->version)) {
```
Comment (laoneo) [inline administrator/components/com_content/src/Model/ArticleModel.php]: If you want to have a one liner, then you can also write it:

`$table->version = empty($table->version) ? 1 : $table->version + 1;`

Comment (Quy) [reply administrator/components/com_content/src/Model/ArticleModel.php]: Thank you!

**Comment by fgsw** [comment]
I have tested this item :white_check_mark: successfully on 6dc12342bc1dacccd077fbc2eb08399af58051e2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44173">issues.joomla.org/tracker/joomla-cms/44173</a>.</sub>

File: administrator/components/com_newsfeeds/src/Model/NewsfeedModel.php [hunk 2]
```diff
@@ -321,10 +322,8 @@ protected function prepareTable($table)
             // Set the values
             $table->modified    = $date->toSql();
             $table->modified_by = $user->get('id');
+            $table->version++;
```
Comment (Quy) [inline administrator/components/com_newsfeeds/src/Model/NewsfeedModel.php]:
```suggestion
            $table->modified_by = $user->id;
            $table->version++;
```

**Comment by Hackwar** [comment]
I have tested this item :white_check_mark: successfully on 0302a85844d3c03488b436c092e0a613c210768d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44173">issues.joomla.org/tracker/joomla-cms/44173</a>.</sub>

**Comment by Hackwar** [comment]
RTC <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44173">issues.joomla.org/tracker/joomla-cms/44173</a>.</sub>

**Comment by pe7er** [comment]
Thanks @Quy !

**Pull request merged**